digdag-standards/src/main/java/io/digdag/standards/operator/PyOperatorFactory.java

```diff
@@ -149,38 +149,19 @@ private Config runCode(final Config state)
                     // Remove the polling state after fetching the result so that the result fetch can be retried
                     // without resubmitting the code.
                     state.remove("commandStatus");
-
-                    StringBuilder reason = new StringBuilder();
-                    reason.append("Python command failed with code ").append(statusCode);
-                    // If the error message and stacktrace are available in outFile,
-                    // throw RuntimeException with them.
+                    String reason = getErrorReason(status, commandContext);
+                    throw new RuntimeException(reason);
```

Comment:
Is it okay if RuntimeException is finally handled as an unexpected exception?

https://github.com/treasure-data/digdag/blob/v0_10/digdag-core/src/main/java/io/digdag/core/agent/OperatorManager.java#L174-L176

Maybe TaskExecutionException is better?

Reply:
The current implementation is as follows:

https://github.com/treasure-data/digdag/blob/v0_10/digdag-standards/src/main/java/io/digdag/standards/operator/PyOperatorFactory.java#L148-L179
```
                if (statusCode != 0) {
...
                    throw new RuntimeException(reason.toString());
                }
```
If status code is not 0, it always throw RuntimeException().
I would like to keep behavior as much as possible.
How about your thought ?

Reply:
I see. Throwing the same exception sounds reasonable.

@muga Just out of curiosity, was there any specific reason to use RuntimeException not TaskExecutionException? https://github.com/treasure-data/digdag/blame/85fd6bdcec06a362f1b16d8de2284168c4e35619/digdag-standards/src/main/java/io/digdag/standards/operator/PyOperatorFactory.java#L149

Reply:
I added ToDo comments on it for future improvements.